superset-frontend/spec/javascripts/views/CRUD/annotation/AnnotationList_spec.jsx

```diff
@@ -97,15 +97,15 @@ describe('AnnotationList', () => {
 
   it('fetches annotation layer', () => {
     const callsQ = fetchMock.calls(/annotation_layer\/1/);
-    expect(callsQ).toHaveLength(3);
-    expect(callsQ[2][0]).toMatchInlineSnapshot(
+    expect(callsQ).toHaveLength(2);
```

Comment:
why did this change the AnnotationList tests? Seems a bit unexpected to me

Reply:
It's this line https://github.com/apache/incubator-superset/pull/11574/files#diff-9c9be0048f0d0d4f1bd547faad9f8732fe5a12df6fc185190681914251131cc9R61 that removed an unecessary network request